### DISCUSSION TITLE
Filesize error despite large upload limit

### DISCUSSION THREAD

**Discussion by wherewego-nina** [payloadcms/payload#595] [General]
# Bug Report

I have an upload collection for images. I have set the upload limit to 5MB just like in the documentation:
`export default buildConfig({
 ...
  upload: {
    limits: {
      fileSize:  5000000 
    },
  },
});`

I then try to upload an image that is 1.4MB and I get the filesize too large error (status 413). I can upload the file locally, but not in production. Also I am not storing the file. I have disabled local storage and I'm using cloudinary to host my images.

## Expected Behavior

I expect to be able to upload an image lower than 5MB

## Current Behavior

I'm not exactly sure but I think the error occurs for anything above ~1MB.

Any idea what's going on? Am I missing something?

**Comment by jmikrut** [comment]
Hey @wherewego-nina — we have seen this before.

So, there are multiple filesize / body size limitations in a typical Node production deployment. Typically, whatever you use as a reverse proxy, in front of your Node process, will enforce its own size limit on requests.

Here is a link that describes how to modify this value with `nginx`:

https://stackoverflow.com/questions/62762025/413-request-entity-too-large-in-digitalocean-nginx

I'd check there. I believe this is going to be entirely outside of Payload's realm though so I'm gonna bump this to a discussion.

What reverse proxy do you use in production?
